### ISSUE TITLE
Optionally allow bullet lists and other indented text in comments when `check-indentation` is enabled

### ISSUE THREAD

**Issue by edg2s** [gajus/eslint-plugin-jsdoc#541]
For example:
```js
/**
 * Foobar
 *
 * This method does the following things:
 * - foo...
 *   this is the first step
 * - bar
 *   this is the second step
 */
```

Rather than looking for bullet lists explicitly, the option could cause the linter to only check the indentation of the first line (of each 'section'?):
```js
/**
 * Allowed
 *   Indentation
 */

/**
 *   Disallowed
 *   Indentation
 */
```

This could also apply to tags with descriptions, instead of having to list them manually:
```js
/**
 * @param {number} val Multi-
 *                     line
 */

/**
 *   @param {number} val Still disallowed
 */
```
